taichi/python/export_ggui.cpp

```diff
@@ -244,6 +245,9 @@ struct PyWindow {
                         vsync,   show_window,        package_path,
                         ti_arch, is_packed_mode};
     // todo: support other ggui backends
+    if (!lang::vulkan::is_vulkan_api_available()) {
```

Comment:
nit: Could you move ` glfwInitVulkanLoader(vkGetInstanceProcAddr);` somewhere inside `PyWindow`? I don't think `glfwInitVulkanLoader(vkGetInstanceProcAddr);` should exist in such a low-level component?

Reply:
I tried to put it in the WindowBase class but in there the `vkGetInstanceProcAddr` is 0x0, it's a bit weird maybe due to symbol visibility.

Reply:
And since GLFW is initialized inside WindowBase's constructor, it needs to happen before that (can't put it inside `vulkan::Window` sadly, that's where it makes most sense, but vulkan::Window is a subclass)

Reply:
> it's a bit weird maybe due to symbol visibility.

I believe this is due to the `volk` initialization process